src/Dotnet.Script.Core/ScriptCompiler.cs

```diff
@@ -15,11 +16,17 @@
 using Dotnet.Script.DependencyModel.Runtime;
 using Microsoft.CodeAnalysis.CSharp.Scripting.Hosting;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using RuntimeAssembly = Dotnet.Script.DependencyModel.Runtime.RuntimeAssembly;
 
 namespace Dotnet.Script.Core
 {
     public class ScriptCompiler
     {
+        // Note: Windows only, Mac and Linux needs something else?
+        [DllImport("Kernel32.dll")]
+        private static extern IntPtr LoadLibrary(string path);
```

Comment:
moving this here, doesn't this affect OmniSharp?

Reply:
No. Omnisharp does not deal with native assets. Those are only needed at runtime. I've simply moved it out of the `RuntimeDependencyResolver` since that class should be about bringing back the runtime dependencies and not about actually loading them. That we do in the `ScriptCompiler` class.